[2021-08-06 08:55:53,331] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: ex_dag_v_1_0.task_a 2021-08-05T00:00:00+00:00 [queued]>
[2021-08-06 08:55:53,351] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: ex_dag_v_1_0.task_a 2021-08-05T00:00:00+00:00 [queued]>
[2021-08-06 08:55:53,352] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-08-06 08:55:53,352] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-08-06 08:55:53,352] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-08-06 08:55:53,359] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): task_a> on 2021-08-05T00:00:00+00:00
[2021-08-06 08:55:53,366] {standard_task_runner.py:52} INFO - Started process 6631 to run task
[2021-08-06 08:55:53,372] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ex_dag_v_1_0', 'task_a', '2021-08-05T00:00:00+00:00', '--job-id', '6', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/ex_dag_v_1_0.py', '--cfg-path', '/tmp/tmpp5_ihaeq', '--error-file', '/tmp/tmphsuu2qt6']
[2021-08-06 08:55:53,373] {standard_task_runner.py:77} INFO - Job 6: Subtask task_a
[2021-08-06 08:55:53,419] {logging_mixin.py:104} INFO - Running <TaskInstance: ex_dag_v_1_0.task_a 2021-08-05T00:00:00+00:00 [running]> on host a3e01b137881
[2021-08-06 08:55:53,479] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Truong
AIRFLOW_CTX_DAG_ID=ex_dag_v_1_0
AIRFLOW_CTX_TASK_ID=task_a
AIRFLOW_CTX_EXECUTION_DATE=2021-08-05T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-08-05T00:00:00+00:00
[2021-08-06 08:55:53,480] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-08-06 08:55:53,481] {subprocess.py:63} INFO - Running command: ['bash', '-c', "echo 'task_a'"]
[2021-08-06 08:55:53,495] {subprocess.py:75} INFO - Output:
[2021-08-06 08:55:53,499] {subprocess.py:79} INFO - task_a
[2021-08-06 08:55:53,499] {subprocess.py:83} INFO - Command exited with return code 0
[2021-08-06 08:55:53,556] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=ex_dag_v_1_0, task_id=task_a, execution_date=20210805T000000, start_date=20210806T085553, end_date=20210806T085553
[2021-08-06 08:55:53,625] {taskinstance.py:1245} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-08-06 08:55:53,670] {local_task_job.py:151} INFO - Task exited with return code 0
